### ISSUE TITLE
Add skip to main content option

### ISSUE THREAD

**Issue by atsansone** [dart-lang/site-www#5795]
Hotlist: https://buganizer.corp.google.com/hotlists/5891206
Bug: https://buganizer.corp.google.com/issues/338062929

### Observed Results

Navigating through the page, noticed that the skip to main content link is missing.

* [Screencast VoiceOver](http://screencast/cast/NTU1Mzc2NjM4NzQxNzA4OHxjMWY4ODRmYi0zYw)
* [Screencast ChromeVox](http://screencast/cast/NjY5OTQzMzgyMjUxOTI5Nnw2NWI2OWQxMS00Yg)

### Expected Results

Ensure that when the user is navigating in the page the skip to main content link should be displayed and work accordingly for going to the main content.

### User Impact

Screen reader and Cognitive users have issues because they cannot bypass repeated headings and blocks of information on the page, they need to navigate everything before reaching the main content.

### Steps To Reproduce

1. Enabled VoiceOver using "Command+F5" or ChromeVox using "Ctrl+Alt+Z".
1. Open the following page:https://dart.dev/guide)
1. Start the linear navigation using "Tab key".
1. Observe that the skip to main content link is not displayed on the page.

### GAR Assessment

[A11yTracker](https://a11ytracker.corp.google.com/products/034ba691-ac78-4cc2-88c2-e06449cfd17e/edit/testresults?useCase=4qqdor1824t28&e=ProductLabels)

### GAR Details

[2.8. Do you provide a bypass for headings and content that are repeated across multiple pages or within the same page?](go/GAR-web#repeated-content)

### Environment Details
* Machine: MAC/ChromeBook
* Browser: Chrome Version 120.0.6099.272 (Official Build) (64-bit)
* Screen reader: VoiceOver/ChromeVox.
* Bug filling time: 15 mins.
